### PR TITLE
Add clarification around <ver> tags in info.xml files.

### PR DESCRIPTION
**Pull request by MikeyMJCO**

### CODE REVIEW

**Comment by mlutfy** [comment]
Looks good. Thanks @MikeyMJCO !

File: docs/extensions/info-xml.md [hunk 1]
```diff
@@ -360,20 +360,21 @@ Elements acceptable within `<urls>`
...
 
-!!! note "Point releases"
-    It is not currently possible to specify compatibility with point releases. If your extension is compatible with CiviCRM 4.7.21 but *not* 4.7.20, then you will need to clearly specify this in the [comments](#comments).
-
-!!! note "Forward compatibility (4.7/5.x)"
-    For CiviCRM 3.x and 4.x, `<ver>` tags must explicitly list all compatible versions.
-
-    For CiviCRM 4.7.x and 5.x, `<ver>` tags imply forward compatibility.
+!!! note "Which version to choose?"
+    Specify the minimum version number required by this extension. It is recommended to use 5.0 whenever possible.
```
Comment (mattwire) [inline docs/extensions/info-xml.md:368]: @MikeyMJCO Not sure about *recommending* 5.0 here as a lot of stuff won't work with 5.0.0! My policy is to specify the last security release (so 5.24).

Comment (MikeyMJCO) [reply docs/extensions/info-xml.md:368]: That's a fair point.

Comment (MikeyMJCO) [reply docs/extensions/info-xml.md]: Removed that part.

**Comment by mattwire** [comment]
Looks good to me now @MikeyMJCO

**Pull request merged**